src/js/view/view.js

```diff
@@ -417,12 +417,16 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
-            if (provider && provider.getName().name.indexOf('flash') === 0) {
+            if (provider &&
+                (provider.getName().name === 'youtube' ||
+                 provider.getName().name === 'chromecast' ||
+                 provider.getName().name.indexOf('flash') === 0 )) {
                 return;
             }
             const title = item.title || '';
             var videotag = _videoLayer.querySelector('video, audio');
-            videotag.setAttribute('title', title);
+            videotag.innerHTML = title;
```

Comment:
@johnBartos @karimJWP though I can't think of a scenario where the video tag has child elements that this code would essentially replace, we should use a dummy div so as not to change the video's `innerHTML`.